test/jdk/java/awt/List/MouseDraggedOriginatedByScrollBarTest.java

```diff
@@ -77,27 +82,52 @@ private static Frame createTestUI() {
             new MouseMotionAdapter(){
                 @Override
                 public void mouseDragged(MouseEvent me){
-                    PassFailJFrame.log(me.toString());
+                    System.out.println(me.toString());
```

Comment:
```suggestion
                    System.out.println(me);
```
`.toString()` is redundant for `println`.

Reply:
Updated all suggestions. Thanks!